### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by samaelFukada** [jan-vandenberg/cruddiy#66]
I'm having trouble in the create section, when I submit, it goes to an error page

**Comment by jan-vandenberg** [comment]
Please send specific errors or a test database.

**Comment by samaelFukada** [comment]
[sertifikasi.sql.zip](https://github.com/jan-vandenberg/cruddiy/files/13426443/sertifikasi.sql.zip)
I used this database, after creating the CRUD application, I tried to insert data, but after pressing the submit button, it was redirected to an error page, but the data was still inserted into the database. 
<img width="1440" alt="Screenshot 2023-11-21 at 19 56 22" src="https://github.com/jan-vandenberg/cruddiy/assets/126968078/a0665208-5c50-47a2-b637-5237652e44f6">

**Comment by germain-italic** [comment]
This error occurs because your table structure has the VARCHAR type as primary keys.

Cruddiy relies on `mysqli_insert_id` to perform redirection after creation, therefore as VARCHAR cannot auto-increment, the last inserted id is undefined.

This kinda relates to #79 too.

I've created a bypass for this situation.

There was also this issue which I think relates to https://github.com/jan-vandenberg/cruddiy/pull/64 :

![](https://files.italic.fr/chrome_evNVahyGJf.png)

I've fixed it:

![](https://files.italic.fr/chrome_cqBLDsMDZN.png)

However when clicking on the button to view the related records, there is another error on the page:

![](https://files.italic.fr/chrome_uwohbimwAj.png)

It looks simple at first sight, but after examining the generated query in entity-index.php, the section "Prepare SQL queries" is messy, maybe due to the use of `CONCAT_WS`. Table names are concatenated multiple times where they shouldn't, and there are errors due to whitespaces. So I'm not touching it atm.
